<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>567</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>8</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>West Coal Auction</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>用户 :</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton_1">
       <property name="text">
        <string>竞价</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>限价</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>执行时间 :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_user"/>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>限定金额 :</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLineEdit" name="lineEdit_limitedamount"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton_test">
     <property name="text">
      <string>连接</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="pushButton_export">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_token"/>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="pushButton_delay">
     <property name="text">
      <string>差时</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton_start">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton_stop">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>结束</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Token :</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QTimeEdit" name="timeEdit">
     <property name="displayFormat">
      <string>h:mm:ss:zzz</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>限价模式金额 :</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>模式 :</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLineEdit" name="lineEdit_orderprice"/>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>日志</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>限价执行毫秒 :</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0" colspan="3">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="15" column="1">
    <widget class="QLineEdit" name="lineEdit_option2_timeout">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>场次 :</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_platid">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>标号 :</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="lineEdit_target">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>可加金额 :</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="lineEdit_addamount"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
